docs/kubernetes.md

```diff
@@ -167,9 +167,150 @@ Max Accuracy: 100.00%
+  name: torch-xla-tpu-2x4
+  labels:
+spec:
+  parallelism: 2 # num of nodes
+  completions: 2 # num of nodes
+  backoffLimit: 0 # default, no retries
+  completionMode: Indexed
+  template:
+    spec:  # pod-spec:
+      serviceAccountName: default
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions: # need to be specified to get tpu resources
+              - key: cloud.google.com/gke-tpu-accelerator
+                operator: "In"
+                values:
+                - "tpu-v5-lite-podslice"
+              - key: cloud.google.com/gke-tpu-topology
+                operator: "In"
+                values:
+                - "2x4"  # 2 nodes of 4 tpu's
+      tolerations:
+      - key: "google.com/tpu"
+        operator: "Equal"
+        value: "present"
+        effect: "NoSchedule"
+      restartPolicy: Never # look in https://kubernetes.io/docs/concepts/workloads/controllers/job/
+      subdomain: headless-svc
+      volumes:
+        # Increase size of tmpfs /dev/shm to avoid OOM.
+      - name: shm
+        emptyDir:
+          medium: Memory
+      containers:
+      - name: training
+        image: us-central1-docker.pkg.dev/tpu-pytorch-releases/docker/xla:nightly_3.10_tpuvm
+        command:
+        - bash
+        - -cxue
+        - |
+          apt-get update -y
+          apt-get install libomp5 numactl libopenblas-dev -y
+          pip3 install mkl mkl-include tf-nightly tb-nightly tbp-nightly numpy
+          
+          ln -s /usr/local/lib/libmkl_intel_ilp64.so.2 /usr/local/lib/libmkl_intel_ilp64.so.1
+
+          mkdir -p pytorch/xla
+          git clone -b r2.3 https://github.com/pytorch/xla.git pytorch/xla
+
+          # Run `args` here
+          "${@:0}"
+        args:
+        - torchrun
+        # Set this to the number of hosts
+        - --nnodes=2
+        # Index provided by Job
+        - --node_rank=$(JOB_COMPLETION_INDEX)
+        # Create one process per local TPU
+        - --nproc_per_node=4
+        # Coordinator always runs on 0th instance of job
+        - --rdzv_endpoint=$(JOB_NAME)-0.headless-svc:12355
+        # Replace this with your script and flags
+        - pytorch/xla/test/test_train_mp_imagenet.py
+        - --model=resnet50
+        - --log_steps=200
+        - --fake_data
+        - --pjrt_distributed
+        - --nometrics_debug
+        - --num_epochs=1
+        ports:
+        - containerPort: 8471  # 8471 is the default port for the TPU VMs communication
+        - containerPort: 12355  # used by the code
+        - containerPort: 8479
+        - containerPort: 8478
+        - containerPort: 8477
+        - containerPort: 8476
+        - containerPort: 8431 # Port to export TPU usage metrics, if supported.
+        volumeMounts:
+        - mountPath: /dev/shm
+          name: shm
+        env:
+        - name: PJRT_DEVICE
+          value: 'TPU'
+        - name: XLA_USE_BF16
```

Comment:
Please remove the `XLA_USE_BF16`. We're removing that option in the next release.

Reply:
done